doc/cascadia/profiles.schema.json

```diff
@@ -325,9 +325,21 @@
             "type": "string"
           },
           "closeOnExit": {
-            "default": true,
-            "description": "When set to true (default), the selected tab closes when the connected application exits. When set to false, the tab will remain open when the connected application exits.",
-            "type": "boolean"
+            "default": "graceful",
+            "description": "When set to graceful/true (default), the selected tab closes when the connected application exits successfully. When set to always, the tab will never be closed automatically. When set to never/false, the tab will remain open when the connected application exits.",
```

Comment:
"When set to always, the tab will never be closed automatically."

isn't this reversed?

Reply:
It absolutely is. Good catch.